external-crates/move/crates/move-compiler/tests/move_2024/parser/global_access_value_invalid.exp

```diff
@@ -7,3 +7,21 @@ error[E01002]: unexpected token
   = Access chains that start with '::' must be one of the following forms: 
       '::<address>::<module>', '::<address>::<module>::<member>'
 
+error[E04007]: incompatible types
+  ┌─ tests/move_2024/parser/global_access_value_invalid.move:4:5
+  │
+2 │     fun foo(): u64 {
+  │                --- Expected: 'u64'
+3 │         1 + ::global_value
```

Comment:
This looks a bit suspicious

Reply:
This is because the function definition is parsed correctly as having the `u64` return type but the body was not parsed correctly and (being treated as empty), typing throws an error on incompatible return type. I adjusted all tests (including this one) to only report parsing errors.